.jenkins/actions/run_standalone.sh

```diff
@@ -126,12 +126,12 @@ if [ "${SAVE_CACHE}" == "true" ] && [ "${SAVE_ARTIFACTS}" == "true" ] ; then
 fi
 rm -rf .gt_cache*
 
-# run analysis and store profiling artifacts
+# store profiling artifacts
 if [ "${DO_PROFILE}" == "true" ] ; then
     echo "Analyzing profiling results"
-    ${BENCHMARK_DIR}/process_profiling.sh
     if [ "${SAVE_ARTIFACTS}" == "true" ] ; then
         echo "Copying profiling information to ${PROFILE_DIR}/prof/"
+        rm -f ${PROFILE_DIR}/prof/*.prof || true
         cp $ROOT_DIR/*.prof ${PROFILE_DIR}/prof/
```

Comment:
why do you need to delete the prof files in PROFILE_DIR / prof before copying the local directory ones there?

Reply:
just to be consistent with what we do in the nsys run (where we remove the `qdrep` file beforehand. This makes it such that we ensure we only ever store one version